pkg/cmd/roachtest/kv.go

```diff
@@ -51,16 +50,18 @@ func registerKV(r *registry) {
 	for _, p := range []int{0, 95} {
 		p := p
 		for _, n := range []int{1, 3} {
-			for _, e := range []bool{false, true} {
-				e := e
-				r.Add(testSpec{
-					Name:  fmt.Sprintf("kv%d/encrypt=%t/nodes=%d", p, e, n),
-					Nodes: nodes(n+1, cpu(8)),
-					Run: func(ctx context.Context, t *test, c *cluster) {
-						runKV(ctx, t, c, p, e)
-					},
-				})
+			minVersion := "2.0.0"
```

Comment:
I think we still want to run two versions when `--encrypt` is true: one with encryption, one without. When false, we would only run the one without encryption. Otherwise, we'll need to run once with encryption on, another off.